components/brave_rewards/browser/publisher_info_database_unittest.cc

```diff
@@ -360,7 +360,8 @@ TEST_F(PublisherInfoDatabaseTest, GetExcludedList) {
    */
   ledger::ActivityInfoFilter filter;
   filter.excluded = ledger::EXCLUDE_FILTER::FILTER_INCLUDED;
-  success = publisher_info_database_->GetActivityList(0, 0, filter, &pub_list);
+  success = publisher_info_database_->GetActivityList(0, 0,
+      filter.Clone(), &pub_list);
```

Comment:
`filter.Clone()` should use `std::move(filter)` as `filter` is no longer used

Reply:
fixed